tools/Common/IceRpc.BuildTelemetry/TelemetryTask.cs

```diff
@@ -56,6 +71,17 @@ protected override string GenerateCommandLineCommands()
         {
             commandLine.AppendSwitch("--contains-slice1");
             commandLine.AppendSwitch(ContainsSlice1);
+            commandLine.AppendSwitch("--contains-slice2");
+            commandLine.AppendSwitch(ContainsSlice2);
+            commandLine.AppendSwitch("--src-file-count");
+            commandLine.AppendSwitch(SourceFileCount);
+            commandLine.AppendSwitch("--ref-file-count");
+            commandLine.AppendSwitch(ReferenceFileCount);
+        }
+        else if (Idl == "Protobuf")
+        {
+            commandLine.AppendSwitch("--src-file-count");
```

Comment:
Rather than add another flag for Protobuf file count (probably `--file-count`), I am just using the `--src-file-count` flag that we use for Slice.